lib/bots/versionbot.ts

```diff
@@ -980,18 +998,9 @@ export class VersionBot extends ProcBot {
 				});
 			}
 		}).then(() => {
-			// Get the labels for the PR.
-			return this.dispatchToEmitter(this.githubEmitterName, {
-				data: {
-					number: pr.number,
-					owner,
-					repo: name
-				},
-				method: this.githubApi.issues.getIssueLabels
-			});
-		}).then((labels: GithubApiTypes.IssueLabel[]) => {
-			// If we don't have a relevant label for merging, we don't proceed.
-			if (_.some(labels, (label) => label.name === MergeLabel)) {
+			// If we don't have a relevant label for merging (or we do but the PR is marked as a WIP),
+			// we don't proceed.
+			if (VersionBot.hasLabel(labels, MergeLabel) && !VersionBot.hasLabel(labels, WIPLabel)) {
```

Comment:
Is it worth documenting that `flow/in-progress` "overrides" `procbots/versionbot/ready-to-merge`, just in case people get confused when adding the `procbots/versionbot/ready-to-merge` label (on a PR that also has the `flow/in-progress` label) doesn't appear to do anything?